### DISCUSSION TITLE
Generate report from CLI with all runs history

### DISCUSSION THREAD

**Discussion by bwomsm1** [allure-framework/allure2#1772] [Questions & Support]
Hi,
I am looking for a way to generate allure web report that will contain all my **previous runs history.** 
I would like to generate that report from CLI 

I would like to create a QA portal that will show the history of our runs for a certain test cases suite. 

What is the best way for doing that? 

Thanks,
Boaz

**Comment by baev** [comment]
did you tried [Allure TestOps](https://qameta.io)?

**Reply by bwomsm1** [reply]
Yes, I know what it is, its just that our team currently don’t have approval for that paid services. 
I am looking for a solution from the free services that found

**Reply by baev** [reply]
then you can check out https://github.com/fescobar/allure-docker-service
